backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/AuthServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package de.bund.digitalservice.ris.caselaw.adapter;
+
+import de.bund.digitalservice.ris.caselaw.domain.AuthService;
+import de.bund.digitalservice.ris.caselaw.domain.DocumentationUnit;
+import java.util.UUID;
+import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthServiceImpl implements AuthService {
+  private final de.bund.digitalservice.ris.caselaw.adapter.AuthService authService;
```

Comment:
Vielleicht können wir die beiden AuthServices unterscheidlich benennen? Oder reicht vielleicht sogar eine Bean mit einem Interface?

Reply:
Es gibt jetzt das AuthService interface (in der domain) und den OAuthService im Adapter, der dieses implementiert. (Naming to be done, aber Ralph meinte, dass im Namen immer stehen soll auf welche Art die Authentifizierung implementiert wird)